### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by solocommand**

### CODE REVIEW

File: src/Asset.php [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+ *
+ * (c) Limit Zero, LLC <contact@limit0.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Limit0\Assets;
+
+use Symfony\Component\HttpFoundation\File\UploadedFile;
```
Comment (zarathustra323) [inline src/Asset.php:14]: Do we need to be dependent on Symfony for this?

Comment (solocommand) [reply src/Asset.php:14]: We could remove it, but a lot of the additional functionality for the `Asset` class comes from File & UploadedFile. Specifically the Mimetype guessing and the setters it does.

Comment (solocommand) [reply src/Asset.php:14]: Actually now that I think about it we can probably cut it out

Comment (zarathustra323) [reply src/Asset.php:14]: It's up to you... would be nice to not need the dep on Symfony. How hard would it be to ditch?

Comment (solocommand) [reply src/Asset.php:14]: @zarathustra323 Looks like it needs to stay for now. Discovered that it wasn't actually storing the ContentType with the S3 call, which is why I initially thought we could remove the dep.

Specifically it just needs the MimeTypeGuesser, but that doesn't appear to be available outside of symfony/http-foundation.

**Pull request merged**